nixos/modules/services/ttys/kmscon.nix

```diff
@@ -41,6 +41,14 @@ in {
           }; in nullOr (nonEmptyListOf fontType);
       };
 
+      useXkbConfig = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc ''
```

Comment:
I think this `lib.mdDoc` causes CI errors. Could you please try to remove it? Or even better test yourself before pushing.

Reply:
Indeed. It turns out there are other keys in `config.services.xserver.xkb` than those we require hence the filter.